src/Expander.hs

```diff
@@ -1072,26 +1074,25 @@ addRootScope' stx = do
   return (addScope' stx rsc)
 
 
-forkExpandOneDecl :: DeclPtr -> Scope -> DeclValidityPtr -> Syntax -> Expand ()
-forkExpandOneDecl dest sc phaseDest stx = do
-  forkExpanderTask $ ExpandDecl dest sc stx phaseDest
```

Comment:
Why are `DeclValidityPtr`s typically named `phaseDest`? My understanding is that a `DeclValidityPtr` gets filled after the name gets bound (but before the thing it gets bound to gets expanded). The expansion of later declarations blocks until all the earlier `DeclValidityPtr`s get filled-in, so we can make sure that all the relevant names are known. Names are related to phases, sure, because the same name can have different meanings in different phases, but that seems like a very minor aspect of a `DeclValidityPtr`. Perhaps that type used to be called `PhasePtr`, and that type later grew to encompass more responsibilities?

Reply:
The reason I called them that when I was writing the code is that a validity pointer represents the place to which I will write information about which phase the name I've just bound is valid in. In other words, the rest of the decls block until they know which phase the previous ones are bound at.

If this was mysterious, please feel free to alpha-convert to something that makes sense to you.

Reply:
I renamed them to things like `declValidityPtr`, `valdityPtr` or `vp`, so the names no longer reference phases; but I added a comment in src/Expander/DeclScope.hs explaining the relationship between DeclValidityPtr and phases.